src/test/suite/markdownUtils.test.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+import { expect } from 'chai';
+import * as mdUtils from '../../markdownUtils';
+
+suite('Markdown Utils Test Suite', () => {
+	
+	test('get the adoc parser', () => {
+		expect(mdUtils.getMDParser()).to.not.be.null;
+	});
+
```

Comment:
what's the link with the sonar reported issues?

Reply:
"Not covered by tests" - https://sonarcloud.io/code?id=vscode-didact&selected=vscode-didact%3Asrc%2FasciidocUtils.ts

Reply:
spinning off to https://github.com/redhat-developer/vscode-didact/pull/294